src/ServiceInsight/Models/Endpoint.cs

```diff
@@ -2,23 +2,31 @@
 {
     public class Endpoint
     {
-        string host;
-
-        public string Name { get; set; }
+        public string Name
+        {
+            get => EndpointDetails.Name;
```

Comment:
This uses expression assignment instead of old get/set style. Not that I don't like it but for consistency, I think we need to keep using the older/established coding style. Same goes for below.

Reply:
Personaly I'd rather see us move everything gradually toward expression bodied members. We should probably be consistent within a single file, though, and I think we are here.

Reply:
Discussed this with @danielmarbach and we think this is better, and it is consistent with other existing members in this file.